docs/music-providers/index.md

```diff
@@ -8,6 +8,7 @@ General Notes:
 - If you remove a provider a cleanup of the database will be done but it takes a little time to complete. If you still see entries from a deleted provider after some time, then try a MA restart to retrigger the cleanup process.
 - Music providers are added by navigating to MA Settings and then clicking on ADD MUSIC PROVIDER at the top of the page
 
-!!! tip "Note" - Depending up on the sequence of adding providers the automatic linking process may need to be initiated again. If what appears to be identical albums or tracks are seen then navigate to the album or track and using the ⋮ menu in the banner at the top of the view select "Refresh Item". This will trigger the linking process and should result in the same albums and tracks being collapsed together.
+!!! tip "Note" 
+    Depending upon the sequence of adding providers, the automatic linking process may need to be initiated again. If what appears to be identical albums or tracks are seen then navigate to the album or track and using the ⋮ menu in the banner at the top of the view select "Refresh Item". This will trigger the linking process and should result in the same albums and tracks being collapsed together.
```

Comment:
this is not good - that is not supposed to be needed

Reply:
I took that from what you said here https://discord.com/channels/753947050995089438/1232522369780682784/1232632962592538708

Reply:
yeah but then create a bug report and lets fix it

Reply:
Ok I have reworded to indicate this is not normal and an issue should be submitted